.gitignore

```diff
@@ -1,4 +1,5 @@
 Infrastructure.pdf
 .DS_Store
 .env
-coverage/
\ No newline at end of file
+coverage/
+server/build/
```

Comment:
Should this have been pushed?

Reply:
Yes, coverage is for coverage reports created by jest, and build is generated when a production build is run.

Reply:
Alright